docs/dev/how-to/send-message-l2-l1.md

```diff
@@ -68,6 +68,8 @@ yarn add zksync-web3 ethers@5 typescript @types/node ts-node
 
 5. Create a `file.ts` file in the root directory with the next script:
 
+Note: Replace `<YOUR_PRIVATE_KEY>` with private key of the wallet to use.
```

Comment:
We should not recommend users type their Private Key in scripts. We should update this script to use the `.env` file to better ensure they don't accidentally share them or commit them into git

Reply:
@MexicanAce fixed by adding instructions for .env file and also updating the scipt
<img width="706" alt="image" src="https://github.com/matter-labs/zksync-web-era-docs/assets/118197764/ee537693-f87f-483a-9b0a-969213007bec">

Reply:
(For step 5 used text from another tutorials to remain docs look solid)